Adds a fidget spinner to toys

Rebruiza opened the pull request:
# This PR is now closed.

mustafakalash commented:
Pls share pic.

mustafakalash commented on code/game/objects/items/toys.dm:
```diff
@@ -837,4 +838,11 @@
+	
+/obj/item/toy/fidgetspinner
+	name = "fidget spinner"
+	desc = "A fidget spinner made out of plasteel.
```
woops

babydoll commented:
compile 👏 before 👏 committing

mustafakalash commented:
Interactable is easy. Just add `/obj/item/toy/fidgetspinner/attack_self()`, and use `visible_message()` and `flick()`.

Crushtoe commented:
To anyone reading: Yes, this is an actual PR. Yes, the intention is to get merged. Yes, it's a bit silly, no, it isn't a total joke.

Sunbeamstress commented:
> tfw you can't stuff the spinner into an autolathe for delicious steel :(

Rebruiza commented:
>tfw you can't stuff the spinner into an autolathe for delicious steel :(
Fixed.

Techhead0 commented:
\>tfw people don't know about escape characters

babydoll commented on code/game/objects/items/toys.dm:
```diff
@@ -837,4 +845,17 @@
+	icon_state = "fidgetspinner"
+	throwforce = 1
+	w_class = ITEM_SIZE_SMALL
+	matter = list(DEFAULT_WALL_MATERIAL = 200)
```
The default wall material is steel, not plasteel

Rebruiza replied:
Description updated.

BlueNexus commented:
WHY

BlueNexus commented:
W H Y

Hubblenaut commented:
Why not adding it to the toy machine?

Crushtoe commented on code/game/objects/items/toys.dm:
```diff
@@ -633,42 +641,42 @@
 	icon_state = "therapyorange"
 	item_state = "egg4" // It's the red one again, lacking an orange item_state and making a new one is pointless
-	w_class = ITEM_SIZE_TINY
+	w_class = ITEM_SIZE_SMALL
```
Undocumented realisticization of toys.

Techhead0 commented on code/modules/client/preference_setup/loadout/loadout_general.dm:
```diff
@@ -96,4 +96,9 @@
+	
+/datum/gear/fidgetspinner
+	display_name = "fidget spinner"
+	description = "A fidget spinner made out of plasteel."
```
Still mentions plasteel here.

ghost commented:
I don't really like this shuriken fidget spinner thing. It just doesn't look serious. Throwing stars already exist, why add throwing stars that resemble a toy?

comma commented:
Gross stale memes, :-1:from mr

dore0062 commented:
Dumb -1

Commit:
Adds a fidget spinner to toys


Fixes accidental comment


Fixes another accidental comment


Tweaks the spinner and other toys


Removes identifying material to allow for a catch-all translation


Fixes another grammar thing

BlueNexus commented:
Dead meme, 👎 (although it'll probably get merged anyway, despite our -1's.)

ghost commented:
Well, the alternative to this would just be general toy grammar fixes and some tweaks.

meyar commented:
Meme PRs are bad and should be discouraged.

Rebruiza commented:
>Meme PRs are bad and should be discouraged.

Really? Please read the disclaimer.

Techhead0 commented:
Neutral. On one hand, it was opened as a meme, not for an honest desire to see fidget spinners exist ingame. (If you want to convince me otherwise, talk to me on IRC/discord.) On the other hand, I kinda want to see some impaled with a fidget spinner.

megaultranuke commented:
Just... Why? Please no.

BlueNexus commented:
@Rebruiza General toy grammar fixes and tweaks are preferable.

mustafakalash commented:
I sincerely doubt that this addition can possibly have any negative impact on the game.

Crushtoe commented:
If this seriously affects you mentally and breaks all immersion, I pray for your recovery.

ghost commented:
@mustafakalash the toy itself won't have any negative impact, but its deadly shuriken version probably will. It doesn't look serious at all. We already have a throwing star item, why add another one?

I'm also pretty sure that many people's immersion and general round experience would be destroyed if they were killed with a fidget spinner, which even sounds stupid.

meyar commented:
Ain't a question of "muh immersion", it's just a stupid, old, dead meme. It does not add value. It's just garbage. It's just bad precedent.

"What do you mean I can't PR this shitty meme, fidget spinners were added" 

And you can say "It's not a meme pr bro!" As much as you like. We're not brain dead.

mustafakalash commented:
Of course, I forgot about our high standard here. We'd rather be killed in reasonable ways, such as being eaten by a roll of tape that a wacky space wizard shot with a magic staff.

ghost commented:
@mustafakalash so you're basically saying that if there are some unreasonable ways to die in the game then we can and have to add even more?

Also, wizard powers don't even correlate with this PR in any way. I'm only pointing out that "shuriken fidget spinner" is a useless and ridiculous item, as we already have an actual throwing star.

Rebruiza commented:
Is the issue the lethal one or the non-lethal one?

ghost commented:
@Rebruiza the lethal one. I'm not against a simple toy being added.

meyar commented:
Pfft, I'd rip out Wizard if I wouldn't _immediately_ be lynched by people for doing so. Bad example.

dore0062 commented:
Poll results (24+ hours:) May change more  in next day or so.

<img width="530" alt="chrome_2017-11-18_14-57-45" src="https://user-images.githubusercontent.com/1994174/32984245-00f9dafe-cc71-11e7-8e8b-c10b2c708e87.png">

Rebruiza commented:
>Pfft, I'd rip out Wizard if I wouldn't immediately be lynched by people for doing so. Bad example.

Not really, since the same applies here. But in this case I didn't abstain by fear of being lynched.

BlueNexus commented:
According to that poll, a lot of people clearly don't want this. Marking as contentious.

ghost commented:
I have no intentions of continuing this PR if it's considered contentious. If everyone is so adamant on keeping this out of the game, then so be it. In the end, it doesn't matter at all for me.

Crushtoe commented:
The fact you made a poll over fidget spinners can only be described as dissapointing and unhealthy.
